src/pages/shipping.js

```diff
@@ -317,60 +286,39 @@ export const boxManifest = async (boxId, userName) => {
     navBarBtn.classList.add('active');
     document.getElementById('contentBody').innerHTML = template;
    
-// We may not need this anymore for data storage:
-    document.getElementById('shippingHiddenTable').innerHTML = JSON.stringify(boxIdAndBagsObj);
-    
-    //addEventNavBarShipment("returnToPackaging");
-    //document.getElementById('boxManifestTable').appendChild(result);
     populateBoxManifestHeader(boxId,boxList,currContactInfo);
     populateBoxManifestTable(boxId, boxIdAndBagsObj, searchSpecimenInstituteArray);
     addEventNavBarShipment("returnToPackaging", userName);
     document.getElementById('printBox').addEventListener('click', e => {
         window.print();
     });
-    addEventNavBarShipment("returnToPackaging", userName);
-    //addEventNavBarShippingManifest();
     hideAnimation();
-    //addEventNavBarShipment("navBarShippingDash");
-    //addEventBackToSearch('backToSearch');
 }
 
 /**
  * 
  * @param {string[]} boxIdArray 
  * @param {string} userName 
  * @param {boolean} isTempMonitorIncluded 
- * @param {*} currShippingLocationNumber 
+ * @param {number} currShippingLocationNumber 
  */
 export const shippingManifest = async (boxIdArray, userName, isTempMonitorIncluded, currShippingLocationNumber) => {
-    let response = await  getBoxes();
+    showAnimation();
+    let response = await getBoxes();
     let boxArray = response.data;
-    let boxIdAndBagsObj = {};
-    let locations = {};
-    let site = '';
+    let siteAcronym = '';
+    let boxIdAndBagsObjToDisplay = {};
 
     for (const box of boxArray) {
-        const boxId= box[conceptIds.shippingBoxId]
-        boxIdAndBagsObj[boxId] = box['bags']
-        locations[boxId] = box[conceptIds.shippingLocation];
-        site = box['siteAcronym'];
-    }
-    
-    let boxIdAndBagsObjToDisplay = {};
-    let location = ''
+        const boxId = box[conceptIds.shippingBoxId];
+        if (!boxIdArray.includes(boxId)) continue;
 
-    for (const boxId of boxIdArray) {
-        boxIdAndBagsObjToDisplay[boxId] = boxIdAndBagsObj[boxId];
-        location = locations[boxId];
+        boxIdAndBagsObjToDisplay[boxId] = box['bags'];
+        !siteAcronym && (siteAcronym = box['siteAcronym']);
```

Comment:
It updates siteAcronym like old code, but only do it once instead of do an update in each iteration. Putting the assignment in the `for` loop may not be the best way, but it was not the focus so I just tweaked the old code.